acceptance_tests/utilities/support_tool_healthcheck.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+from time import sleep
+
+from requests.exceptions import ConnectionError, HTTPError
+
+from acceptance_tests.utilities.iap_requests import make_request
+
+
+def support_tool_healthcheck(url: str, max_retries: int):
+    check_support_tool = True
+    num_of_retries = 0
+    while check_support_tool:
+        try:
+            response = make_request(url=url)
+            response.raise_for_status()
+            check_support_tool = False
+        except (ConnectionError, HTTPError):
+            num_of_retries += 1
+            if num_of_retries == max_retries:
+                print("Reached maximum number of retries. Stopping script")
+                exit(1)
+
+            print("Support Tool not available. Sleeping for a minute")
+            sleep(60)
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser(
+        description="This script polls the Support Tool health endpoint to check if it's available for the Acceptance "
+                    "Tests to start running")
```

Comment:
Apart from the variable naming, this script is already completely generic. Is it worth leaning into that and making the naming all generic so we can easily repurpose it, for the upcoming self-service UI for example?

Also, maybe this could live in the top level directory for visibility since it's supposed to be run directly, not called by test code.

Reply:
I leant more into the support tool naming since I didn't know if it would be used for anything else. I can make it more generic no problem.

Shouldn't be a problem putting it in the top level!